src/rn2xx3.cpp

```diff
@@ -195,6 +212,38 @@ bool rn2xx3::initOTAA(String AppEUI, String AppKey)
   return joined;
 }
 
+
+bool rn2xx3::initOTAA(uint8_t * AppEUI, uint8_t * AppKey, uint8_t * DevEUI)
```

Comment:
I've done this one because I'm storing keys in Flash memory as byte array, I do not use then hard coded because each time I'm flashing and testing node I need to change depending on node number. I'm flashing them one time then I can upload new sketch to my node without worry on ID and my ID are not on code when I push to github;-)